Libraries/Components/SafeAreaView/SafeAreaView.js

```diff
@@ -31,24 +29,33 @@ let exported;
  * sensor housing area on iPhone X).
  */
 if (Platform.OS === 'android') {
+  let exported;
   exported = class SafeAreaView extends React.Component<Props> {
```

Comment:
Can you use forwardRef for this one too? We’d want to be consistent on the exported interface of the module.

Reply:
Emm, actually, we don't have `SafeAreaView` for Android.

Reply:
Yeah. What I mean is that we render a View on Android, it seems like we should forward the ref onto the View that is rendered instead. What do you think?